### DISCUSSION TITLE
自己数据集训练效果很差

### DISCUSSION THREAD

**Discussion by h4444433333** [PaddlePaddle/PaddleOCR#12767] [Q&A]
#### 问题描述 / Problem Description
我对细长图的检测训练效果很差，图片内有两行文字，一行44个字符，内容为身份识别码。我v3和v4 都尝试过，config文件v3
用的det_mv3_db,都用的默认配置。一个识别不出来，infer会预测多很多框，但是都不对，正确的应该是预测出两个框，分别框第一行和第二行，用原生的ppv4的det权重效果就很好。

#### 运行环境 / Runtime Environment
- OS:linux
- Paddle:最新版本
- PaddleOCR:最新版本

**Comment by SWHL** [comment]
猜测自己构建数据集数量太少了，增加数据集规模试试
